externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -185,12 +194,13 @@ object SparkSQLEngine extends Logging {
             case _ => error("Current SparkSQLEngine is not created.")
           }
 
-      }
-    } catch {
-      case t: Throwable => error(s"Failed to instantiate SparkSession: ${t.getMessage}", t)
-    } finally {
-      if (spark != null) {
-        spark.stop()
+        }
+      } catch {
+        case t: Throwable => error(s"Failed to instantiate SparkSession: ${t.getMessage}", t)
+      } finally {
+        if (spark != null) {
+          spark.stop()
```

Comment:
The `spark` could be set to null.

Reply:
> The `spark` could be set to null.

Sorry, I don't understand what this needs to do.